src/features/defi/providers/idle/components/IdleManager/Overview/IdleOverview.tsx

```diff
@@ -236,6 +243,17 @@ export const IdleOverview: React.FC<IdleOverviewProps> = ({
 
   if (!underlyingAssetsWithBalancesAndIcons || !opportunityData) return null
 
+  if (bnOrZero(fiatAmountAvailable).eq(0) && !hideEmptyState) {
+    return (
+      <IdleEmpty
```

Comment:
Missing a `key` property.

Reply:
Not sure I follow what needs a key here?